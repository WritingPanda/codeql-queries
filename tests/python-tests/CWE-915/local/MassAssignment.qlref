CWE-915/MassAssignmentLocalProblem.ql