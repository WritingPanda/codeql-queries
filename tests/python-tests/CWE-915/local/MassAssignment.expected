| massassignment.py:18:26:18:28 | ControlFlowNode for key | Use of $@. | massassignment.py:30:10:30:20 | ControlFlowNode for Attribute | mass assignment |
| massassignment.py:24:15:24:25 | ControlFlowNode for Attribute | Use of $@. | massassignment.py:24:15:24:25 | ControlFlowNode for Attribute | mass assignment |
| massassignment.py:27:18:27:28 | ControlFlowNode for Attribute | Use of $@. | massassignment.py:27:18:27:28 | ControlFlowNode for Attribute | mass assignment |
